unlock-app/src/components/content/LogContent.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Head from 'next/head'
+import Layout from '../interface/Layout'
+import Account from '../interface/Account'
+import BrowserOnly from '../helpers/BrowserOnly'
+import GlobalErrorConsumer from '../interface/GlobalErrorConsumer'
+import CreatorLog from '../creator/CreatorLog'
+import { pageTitle } from '../../constants'
+import withConfig from '../../utils/withConfig'
+import * as UnlockTypes from '../../unlock'
+
+interface Props {
+  account: UnlockTypes.Account
+  network: UnlockTypes.Network
+  transactionFeed: UnlockTypes.Transaction[]
+  explorerLinks: { [key: string]: string }
+}
+
+export const LogContent = ({ account, network, transactionFeed, explorerLinks }: Props) => {
+  return (
+    <GlobalErrorConsumer>
+      <Layout title="Creator Log">
+        <Head>
+          <title>{pageTitle('Log')}</title>
+        </Head>
+        {account && (
+          <BrowserOnly>
+            <Account network={network} account={account} />
+            <CreatorLog transactionFeed={transactionFeed} explorerLinks={explorerLinks} />
+          </BrowserOnly>
+        )}
+      </Layout>
+    </GlobalErrorConsumer>
+  )
+}
+
+interface State {
```

Comment:
I wonder if we can find a more specific name for this and `Config`? Let's open up a follow-up issue and discuss there?

Reply:
I'm definitely open to suggestions, note that these types never leave this file though. They're really just there to keep `no-any` off our backs.